### DISCUSSION TITLE
Release Date of the Next WinUI/WASDK Version Exp 3 or Preview 1

### DISCUSSION THREAD

**Discussion by ghost1372** [microsoft/microsoft-ui-xaml#9878] [Q&A] [answered]
Hi @codendone ,
I have noticed that it has been approximately 1 Month + three weeks since the last experimental version of WinUI/WASDK was released. Could you please provide information on when the next version is scheduled to be released? Exp 3 or preview 1?

Thank you for your time and assistance.

**Comment by ghost1372** [comment]
@duncanmacmichael any news ?

**Comment by duncanmacmichael** [comment, marked as answer]
Hi @ghost1372, thanks for the question! 1.6-preview1 is releasing today; I'm working on the release notes now. Stay tuned! :)
